HISTORY.rst

```diff
@@ -4,7 +4,7 @@ History
 
 0.3.0 (2018-01-09)
```

Comment:
should we maybe update to 0.3.1?

Reply:
yeah, with the next release. in history it's always the last one on PyPI, which is in fact 0.3.0. The next one with this fix is 0.3.1